composer.json

```diff
@@ -29,12 +29,17 @@
         {
             "type": "vcs",
             "url": "https://github.com/JellyBellyDev/phpspec-data-provider-extension"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/ricardoboss/easysvg"
```

Comment:
do we really need to depend on the fork of the third party lib?

Reply:
What do you mean by unpublished?

I specifically explained why this fork has a release: https://github.com/ricardoboss/easysvg/releases/tag/2.0

We need this fork, since the original library has a lot of deprecation warnings (and even fatal errors) because it supported very old PHP versions. In my fork, I fixed them and opened a PR. I am in talks with the creator of the library to merge my PR and then we can drop this extra repository.

Reply:
Nice! I pinged https://github.com/kartsims/easysvg/pull/32#issuecomment-1038998987 :)